sdk/digitaltwins/Azure.DigitalTwins.Core/src/Azure.DigitalTwins.Core.csproj

```diff
@@ -11,7 +11,7 @@
   <PropertyGroup>
     <PackageTags>IoT;DigitalTwins;$(PackageCommonTags)</PackageTags>
     <Description>SDK for the Azure Digital Twins service</Description>
-    <Version>1.3.0-beta.1</Version>
+    <Version>1.2.2</Version>
```

Comment:
Check the AssemblyInfo.cs file too.

Reply:
No mention of the version in `AssemblyInfo.cs` file.